charts/gha-runner-scale-set-controller/tests/template_test.go

```diff
@@ -404,6 +404,8 @@ func TestTemplate_ControllerDeployment_Customize(t *testing.T) {
 		SetValues: map[string]string{
 			"labels.foo":                   "bar",
 			"labels.github":                "actions",
+			"labels.team":                  "GitHub Team",
+			"labels.teammail":              "team@github.com",
```

Comment:
you might want to also assert these on line 449, similar to `assert.Equal(t, "actions", deployment.Labels["github"])`

Reply:
Aw man, I did not read further than I should have. Sorry, added assert statements for both cases now.